datagenerator/src/main/resources/rythm/enum.rythm

```diff
@@ -0,0 +1,48 @@
+@import com.linkedin.restli.datagenerator.csharp.CSharpEnum;
+@import com.linkedin.restli.datagenerator.csharp.CSharpRythmGenerator;
+@import org.rythmengine.RythmEngine
+@args CSharpEnum cSharpEnum, CSharpRythmGenerator generator, RythmEngine engine
+using restlicsharpdata.restlidata;
+using System;
+
+// DO NOT EDIT - THIS FILE IS GENERATED BY restli-csharp@cSharpEnum.restli_generatedFrom()
+
+namespace @cSharpEnum.getNamespace()
+{
+  @cSharpEnum.getDoc().restli_comment()
+  public class @cSharpEnum.getName() : EnumTemplate
```

Comment:
It seems that some of your generated files are not newly generated based on this rythm, some are not extended from `EnumTemplate`. Same for other generated record, and union. After fixing rythm files, please always regenerate all the files.